build/nsis/Mixxx.nsi

```diff
@@ -420,6 +420,7 @@ Section "Uninstall"
   Delete "$INSTDIR\controllers\Hercules DJ Control MP3.hid.xml"
   Delete "$INSTDIR\controllers\Hercules DJ Control MP3.midi.xml"
   Delete "$INSTDIR\controllers\Hercules DJ Control Steel.midi.xml"
+  Delete "$INSTDIR\controllers\Hercules-DJ-Console-4-Mx-scripts.js"
```

Comment:
It's already there -- this was a pre-existing preset but it used the Mk4 scripts.